### DISCUSSION TITLE
Lidarr Script is running into errors during "installing Lidarr" stage

### DISCUSSION THREAD

**Discussion by Joshhorn4** [tteck/Proxmox#3918] [Q&A]
Long time listener first time caller here, 

Would anybody be able to help me resolve this issue when using the Lidarr install script please,

[ERROR] in line 23: exit code 0: while executing command "$@" > /dev/null 2>&1
The silent function has suppressed the error, run the script with verbose mode enabled, which will provide more detailed output.

Script used 

bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/lidarr.sh)"

![Screenshot 2024-10-18 at 11 38 44 am](https://github.com/user-attachments/assets/0624a248-10ba-4c04-b90d-ffda440d145c)
.

**Comment by tteck** [comment]
> The silent function has suppressed the error, run the script with verbose mode enabled, which will provide more detailed output.

Step one.

**Reply by Joshhorn4** [reply]
re ran in verbose and recieved this 

/--2024-10-18 14:33:23--  https://lidarr.servarr.com/v1/update/master/updatefile?os=linux&runtime=netcore&arch=x64
Resolving lidarr.servarr.com (lidarr.servarr.com)... 104.21.3.105, 172.67.130.145, 2606:4700:3036::ac43:8291, ...
Connecting to lidarr.servarr.com (lidarr.servarr.com)|104.21.3.105|:443... connected.
 |404 Not Foundsent, awaiting response... 
2024-10-18 14:33:24 ERROR 404: Not Found.


[ERROR] in line 28: exit code 0: while executing command $STD wget --content-disposition 'https://lidarr.servarr.com/v1/update/master/updatefile?os=linux&runtime=netcore&arch=x64'
root@blank2:~#

**Reply by tteck** [reply]
They're having issues with `servarr.com`
https://github.com/tteck/Proxmox/issues/3915

**Reply by Joshhorn4** [reply]
That would explain it, thanks mate :)
